src/app.js

```diff
@@ -95,3 +99,18 @@ setInterval(function() {
 panel.on('click', 'select', updatePokemon);
 panel.on('longClick', 'select', function() { options.debug = !options.debug; updatePokemon(); });
 
+Pebble.addEventListener('showConfiguration', function(e) {
+  Pebble.openURL(clay.generateUrl());
+});
+
+Pebble.addEventListener('webviewclosed', function(e) {
```

Comment:
Ah yeah, that's perfect! I'll move it out of app.js and add an updateTheme function to the relevant files :)